server/etcdmain/etcd.go

```diff
@@ -57,6 +57,8 @@ func startEtcdOrProxyV2(args []string) {
 
 	err := cfg.parse(args[1:])
 	lg := cfg.ec.GetLogger()
+	cfg.ec.SetupGlobalLoggers()
```

Comment:
Nit: What do you think, should this maybe be moved before getting the logger, so to line 59:
```
	cfg.ec.SetupGlobalLoggers()
	lg := cfg.ec.GetLogger()
```

Reply:
I moved it to the later place of the file, so make it just after error handling (see the other comment)